wp-convertkit.php

```diff
@@ -13,6 +13,8 @@
  * Author: Kit
  * Author URI: https://kit.com/
  * Text Domain: convertkit
+ * License:     GPLv2 or later
```

Comment:
Aren't we GPL 3?

Reply:
It's always been GPLv2 according to the [readme file](https://github.com/Kit/convertkit-wordpress/blob/main/readme.txt#L9)

Have updated both to v3.